File: articles/research-frameworks-and-analytical-models/c/framework_summary.c
#include <stdio.h>

int main(void) {
    int total_elements = 9;
    int documented_elements = 6;
    double coverage = (double) documented_elements / (double) total_elements;

    printf("Research framework element coverage: %.3f\n", coverage);
    return 0;
}

File: articles/research-frameworks-and-analytical-models/fortran/framework_summary.f90
program framework_summary
  implicit none
  integer :: total_elements
  integer :: documented_elements
  real :: coverage

  total_elements = 9
  documented_elements = 6

  coverage = real(documented_elements) / real(total_elements)

  print *, "Research framework element coverage:", coverage
end program framework_summary
